src/manual-edits/content/docs/new-relic-only/basic-style-guide/writing-guidelines/view-docs-revision-history-diff.mdx

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
How will this work now?? Is this doc valid any more?

Reply:
Yeah, I deleted this doc after moving it.